clients/da-vinci-client/src/test/java/com/linkedin/davinci/client/StatsAvroGenericDaVinciClientTest.java

```diff
@@ -42,8 +41,8 @@ public void testGet(boolean reuseApi) throws ExecutionException, InterruptedExce
     when(mockClient.getStoreName()).thenReturn(storeName);
 
     VeniceMetricsRepository metricsRepository = getVeniceMetricsRepository(
-        DAVINCI_CLIENT_SERVICE_NAME,
-        DAVINCI_CLIENT_METRIC_PREFIX,
+        DAVINCI_CLIENT.getName(),
+        DAVINCI_CLIENT.getMetricsPrefix(),
```

Comment:
This looks good! Since we add this new `ClientType`, I am thinking can we overload `getVeniceMetricsRepository` so that the interface could be simplified and only takes a `ClientType` reference, instead of two Strings? I promise this is the last nit comment from me :-)

Reply:
Cleaned it up. Thanks.